cmake/projects/OpenSSL/schemes/url_sha1_openssl_windows_1_1_plus.cmake.in

```diff
@@ -99,7 +112,14 @@ ExternalProject_Add(
     CONFIGURE_COMMAND
     "@HUNTER_MSVC_VCVARSALL@" "@HUNTER_MSVC_ARCH@"
     COMMAND
-    perl Configure "${opt}" ${shared} no-asm "--prefix=@HUNTER_PACKAGE_INSTALL_PREFIX@" "--openssldir=${openssl_dir}"
+        "@CMAKE_COMMAND@"
+        -E copy
+        "@HUNTER_PACKAGE_BUILD_DIR@/nasm_path_configure.cmake"
+        "${openssl_dir}/nasm_path_configure.cmake"
```

Comment:
`openssl_dir` is not the right choice here because files from this directory will be installed, `nasm_path_configure.cmake` is an internal file.

Reply:
which path should I use ?

Reply:
`@HUNTER_PACKAGE_SOURCE_DIR@` should be fine.